Automattic/sensei
# Add the email preview

Part of #6465

### Changes proposed in this Pull Request

* Add an email preview screen that shows the email subject, sender, and body.
* I wasn't able to add the "Preview" link on the post edit screen, so I've added it to the email listing table following the same convention as WP (see screenshot). 

I will attempt to add the "Preview" link to the post email screen in a separate [PR](https://github.com/Automattic/sensei/pull/6583).

### Testing instructions

* Enable the feature flag: `add_filter( 'sensei_feature_flag_email_customization', '__return_true' );`
* Go to Sensei LMS -> Settings -> Emails.
* Have an email post with placeholders.
* Hover over an email post and click the "Preview" link.
* A preview of the email should be presented with the placeholders replaced.

### Screenshot / Video

![image](https://user-images.githubusercontent.com/1612178/221860809-83d84055-ad9b-4398-bf1d-67c391debc0f.png)

![image](https://user-images.githubusercontent.com/1612178/221324709-d36aa4d4-131e-48fa-93c6-43d0d7512f81.png)
